### PR TITLE
Fix to work on Node >=4, add minimal test

### PR DESCRIPTION
**Pull request by rattrayalex-stripe**
Heya James,

Nice work here, I had a `flow-watch` built with `blessed` I was getting ready to release, but this looks way better. 

package.json's engine specified node >=4, but on node 6.5 it crashed looking for util.promisify, which was added in node 8. 

I tested by doing `nvm install 4 --lts` and then, on `v4.8.7`, running `yarn test`. Tests passed. 

Cheers!

### CODE REVIEW

**Comment by rattrayalex-stripe** [comment]
Note that I'm probably over my "fiddle with open source" budget for the day, so would probably have to decline/delay any cosmetic code review unfortunately.

File: test.js.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The actual snapshot is saved in `test.js.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## runs without crashing
+
+> Snapshot 1
+
+    `🕵️‍♀️  Glow v1.1.0 (flow-bin v0.63.1)␊
```
Comment (thejameskyle) [inline test.js.md]: This snapshot is going to break whenever a new version of Glow is released.

Could just just assert that the stdout contains a message?

Comment (rattrayalex-stripe) [reply test.js.md]: Yeah good call, will do.

**Comment by rattrayalex-stripe** [comment]
Updated. 

Might be a good idea to get a Travis CI build running on Node 4 by the way. Don't think I have the bandwidth to help with this right now unfortunately.

**Comment by thejameskyle** [comment]
Manually merged

**Pull request closed without merging**